api/server/router/network/network_routes.go

```diff
@@ -412,3 +357,82 @@ func (n *networkRouter) postNetworksPrune(ctx context.Context, w http.ResponseWr
 	}
 	return httputils.WriteJSON(w, http.StatusOK, pruneReport)
 }
+
+func (n *networkRouter) findNetwork(term string) (types.NetworkResource, error) {
+	// NOTE: This findNetwork is differnt from FindNetwork from the daemon.
```

Comment:
Could you make this comment a docstring (above the function) ? Right now it looks like it's documenting the variables inside the function.

Also what do you think about `findUniqueNetwork()` as a name?